sdk/communication/azure-communication-common/src/test/java/com/azure/android/communication/common/CommunicationTokenCredentialTest.java

```diff
@@ -212,17 +207,18 @@ public void getToken_onDemandAutoRefresh_noInitialToken_firstFetch_multithreaded
         // Set up blocked multithreaded calls
         Runnable blockedRefresh = this.arrangeBlockedRefresh(mockTokenRefresher);
         int numCalls = 5;
-        List<Future<AccessToken>> accessTokenFutures = new ArrayList<>();
+        List<Future<CommunicationAccessToken>> accessTokenFutures = new ArrayList<>();
```

Comment:
CompletableFuture here instead of Future. Or is there a particular reason why we want to keep as Future?

Reply:
I see instances of Future all throughout the file as well

Reply:
yes, `CompletableFuture` is a `Future`, so unless we're exercising specific apis from CF in the tests, its totally fine to use receiving type as `Future`. But thinking a bit more, I feel that it's a good idea to use concrete type, just to catch any breaking in return type, will update the PR to `CF`.